README.md

```diff
@@ -108,7 +108,7 @@ jobs:
       # Run breaking change detection against the last commit
       - uses: bufbuild/buf-breaking-action@v1
         with:
-          against: 'https://github.com/acme/weather.git#branch=main,ref=HEAD~1'
+          against: 'https://github.com/${GITHUB_REPOSITORY}.git#branch=main,ref=${GITHUB_SHA}~1'
```

Comment:
I think this would fail for `pull_request` events as the `GITHUB_SHA` is the last puhsed to the current branch: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
Could you validate this change works for `push` and `pull_request` event types?